packages/astro-theme/layouts/ProductHome.astro

```diff
@@ -6,6 +6,7 @@ import BentoGrid from "../components/patterns/BentoGrid.astro";
 import { productFromUrl } from "../lib/products";
 
 const product = productFromUrl(Astro.url)!;
+const {style = "px-6 py-10 w-full  mx-auto bg-gradient-to-b to-[#ffffff44] from-stone-900"}= Astro.props;
```

Comment:
I'd probably make all of these classes default and merge in a `class` prop using clsx